dev/source/docs/custom-build-server.rst

```diff
@@ -0,0 +1,81 @@
+.. _custom-build-server:
+
+=================================
+ArduPilot Custom Firmware Builder
+=================================
+
+As the ArduPilot software evolves through community contributions and development efforts, it’s expanding rapidly with many new features being added. However, not all users need every feature. Depending on their specific requirements and applications, users might prefer to exclude certain features from the software.
+
+To address this need, we have developed the **Custom Firmware Builder**, a web application hosted at `custom.ardupilot.org <https://custom.ardupilot.org/>`__. This process helps in reducing the firmware size, making it easier to fit into flight control units (FCUs) with limited flash memory. It also benefits our testers, allowing them to test specific features without needing to upgrade their flight controllers due to space constraints.
+
+The source code of the custom firmware builder can be found `here. <https://github.com/ArduPilot/CustomBuild/>`__
+
+Submitting a build request
+--------------------------
+
+To submit a build request, follow these steps:
+
+#. Click the **'Add a build'** button located at the top right corner of the homepage.
+#. Choose the vehicle, version, and board for which you want to build the firmware.
```

Comment:
```suggestion
#. Choose the vehicle, version, and board ,in that order, for which you want to build the firmware.
```
order is important since it resets the bd if vehicle changes...right?

Reply:
Right